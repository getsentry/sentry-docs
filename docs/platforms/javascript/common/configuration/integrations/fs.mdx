---
title: FileSystem
description: "Adds instrumentation for filesystem operations."
supported:
  - javascript.node
  - javascript.aws-lambda
  - javascript.azure-functions
  - javascript.connect
  - javascript.express
  - javascript.fastify
  - javascript.gcp-functions
  - javascript.hapi
  - javascript.koa
  - javascript.nestjs
  - javascript.electron
  - javascript.nextjs
  - javascript.nuxt
  - javascript.sveltekit
  - javascript.remix
  - javascript.astro
  - javascript.bun
---

<Alert level="info">

This integration only works in the Node.js and Bun runtimes.

</Alert>

_Import name: `Sentry.fsIntegration`_

The `fsIntegration` creates spans for `fs` API operations, like reading and writing files. The integration uses the [`@opentelemetry/instrumentation-fs`](https://www.npmjs.com/package/@opentelemetry/instrumentation-fs) package.

<Alert level="warning" title="Potential Performance Overhead">
  The `fsIntegration` may add significant overhead to your application.
  Especially in scenarios with a lot of file I/O, like for example when you are
  running a framework dev server, including this integration can massively slow
  down your application.
</Alert>

## Options

### `recordFilePaths`

_Type: `boolean | undefined`_

Setting this option to `true` will include any filepath arguments from your `fs` API calls as span attributes.
Defaults to `false`.

### `recordErrorMessagesAsSpanAttributes`

_Type: `boolean | undefined`_

Setting this option to `true` will include the error messages of failed `fs` API calls as a span attribute.
Defaults to `false`.
