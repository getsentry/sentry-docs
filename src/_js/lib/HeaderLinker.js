$(document).on('page.didUpdate', function() {
  $('h1,h2,h3,h4,h5,h6').each((i, el) => {
    const $el = $(el);
    const id = $el.attr('id');
    if(!id) {
      return;
    }

    // This event gets also fired whenever history.replaceState is called. In
    // that case we do not want to prepend multiple anchors.
    if ($('.anchor', $el).length > 0) {
      return;
    }

    const $link = $(`
      <a class="anchor" aria-hidden="true" href="#${id}">
        <svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M10.879 6.05L15 1.93A5.001 5.001 0 0 1 22.071 9l-4.121 4.121a1 1 0 0 1-1.414-1.414l4.12-4.121a3 3 0 1 0-4.242-4.243l-4.121 4.121a1 1 0 1 1-1.414-1.414zm2.242 11.9L9 22.07A5 5 0 1 1 1.929 15l4.121-4.121a1 1 0 0 1 1.414 1.414l-4.12 4.121a3 3 0 1 0 4.242 4.243l4.121-4.121a1 1 0 1 1 1.414 1.414zm-8.364-.122l13.071-13.07a1 1 0 0 1 1.415 1.414L6.172 19.242a1 1 0 1 1-1.415-1.414z" fill="currentColor"/></svg>
      </a>
    `);
    $el.prepend($link);
  });
});
