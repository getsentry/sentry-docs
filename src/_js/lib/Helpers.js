export const fullPath = function({ pathname: target, hash, search }) {
  return `${target}${search || ''}${hash || ''}`;
};
