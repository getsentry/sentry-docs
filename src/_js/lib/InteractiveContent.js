// These are the variables that can be replaced in the document
export const tokens = [
  'DSN',
  'PUBLIC_DSN',
  'PUBLIC_KEY',
  'SECRET_KEY',
  'API_URL',
  'ENCODED_API_KEY',
  'PROJECT_ID',
  'ORG_NAME',
  'PROJECT_NAME',
  'MINIDUMP_URL'
];

// Helper to generate a regex from the approved tokens
const tokenRegex = function() {
  var str = `___(${tokens.join('|')})___`;
  var re = new RegExp(str, 'g');
  return re;
};

const slugify = function(input) {
  return input
    .toLowerCase()
    .replace(/___/g, '')
    .replace(/_/g, '-');
};

// Wrap all instances of ___VARIABLE___ with a span we can use for easy updates
const wrapTokens = function($src) {
  const content = $src.html().replace(tokenRegex(), function(match) {
    const slug = slugify(match);
    const kind = slug === 'encoded-api-key' ? 'key' : 'dsn';
    const list = User.userData[`${kind}List`];
    const grouped = list.reduce((obj, item) => {
      if (obj[item.group] === undefined) obj[item.group] = [];
      obj[item.group].push(item);
      return obj;
    }, {});

    return `
      <span class="dropdown mb-1" data-toggle="tooltip" data-placement="left" title="Showing example configuration">
        ${renderDropdownTarget({ kind, slug, match })}

        <div class="dropdown-menu family-sans" aria-hidden="true">${Object.keys(
          grouped
        )
          .map(key => {
            return `${renderDropdownHeader(key)}${grouped[key]
              .map(renderDropdownItem)
              .join('|')}`;
          })
          .join('')}</div>
      </span>
    `.replace(/(\n|\s{2})/g, '');
  });
  $src.html(content);
  $('[data-toggle="tooltip"]').tooltip('enable');
};

const renderDropdownTarget = function({ kind, slug, value }) {
  return `<a data-interactive data-kind="${kind}" data-key="${slug}" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">${value}</a>`;
};

const renderDropdownItem = function(item) {
  return `<a href="#" class="dropdown-item" data-interactive-content-item data-id="${
    item.id
  }" data-kind="${item.kind}">${item['project-name']}</a>`;
};

const renderDropdownHeader = function(item) {
  return `<h6 class="dropdown-header">${item}</h6>`;
};

const renderDropdown = function(list) {
  if (!list) return;
  let current = list[0];
  const grouped = list.reduce((obj, item) => {
    if (obj[item.group] === undefined) obj[item.group] = [];
    obj[item.group].push(item);
    return obj;
  }, {});

  return `
    <div class="dropdown mb-1">
      <a class="btn btn-sm btn-secondary dropdown-toggle" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
        ${current.name}
      </a>

      <div class="dropdown-menu" aria-hidden="true">${Object.keys(grouped)
        .map(key => {
          return `${renderDropdownHeader(key)}${grouped[key]
            .map(renderDropdownItem)
            .join('|')}`;
        })
        .join('')}</div>
    </div>
  `;
};

const setActiveId = function(kind, id) {
  localStorage.setItem(`${kind}Preference`, id);
};

const getData = function(kind) {
  const id = localStorage.getItem(`${kind}Preference`) || -1;
  const list = User.userData[`${kind}List`];
  const data = list.find(obj => parseInt(obj.id) === parseInt(id));
  return data;
};

// Updates all tokens to show values based on the values stored in localStorage
const updateTokens = function() {
  $('[data-interactive]').each(function(i, el) {
    const $el = $(el);
    const key = $el.data('key');
    const kind = $el.data('kind');
    const data = getData(kind);
    $el.html(unescape(data[key]));
    $el.closest('.dropdown').attr('data-original-title', data['project-name']);
  });
};

const init = function() {
  $(document).on('user.ready', function(event) {
    const $main = $('main').eq(0);
    const hasInteractiveContent = tokenRegex().test($main.html());
    if (hasInteractiveContent) {
      wrapTokens($main);
      updateTokens();
    }

    $(document).on('click', '[data-interactive-content-item]', function(event) {
      const $el = $(event.target);
      setActiveId($el.data('kind'), $el.data('id'));
      updateTokens();
    });

    $(document).trigger('dynamicContent.didLoad');
  });
};

export default { init };
