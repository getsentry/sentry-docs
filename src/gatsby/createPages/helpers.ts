export const getChild = (node: any) => {
  return node.childMarkdownRemark || node.childMdx;
};

export const getDataOrPanic = async (query, graphql, reporter) => {
  const { data, errors } = await graphql(query);
  if (errors) {
    reporter.panicOnBuild(`ðŸš¨  ERROR: ${errors}`);
  }
  return data;
};
