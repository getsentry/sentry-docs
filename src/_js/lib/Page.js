import { canonicalPath, canonicalUrl } from './Helpers';

export const logGooglePageview = function() {
  if (!window.gtag) return;

  const i = location.href.indexOf('#');
  const url = i === -1 ? location.href : location.href.slice(0, i);
  const payload = {
    anonymize_ip: true,
    page_title: document.title,
    page_location: url,
  };
  window.gtag('config', 'UA-30327640-1', payload);
  window.gtag('config', 'UA-30327640-3', payload);
};

export const logPageview = function() {
  window.amplitude.getInstance().logEvent('Viewed Docs Page', {
    path: canonicalPath(),
    referrer: document.referrer,
    search: location.search,
    title: document.title,
    url: canonicalUrl(location.search)
  });
  window.ra.page();
  logGooglePageview();
};
