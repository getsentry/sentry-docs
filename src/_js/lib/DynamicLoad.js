import { logPageview } from './Page';
import { fullPath } from './Helpers';

const isDifferentDomain = function(here, there) {
  return here.protocol !== there.protocol || here.host !== there.host;
};

const isSamePage = function(here, there) {
  return (
    here.pathname.replace(/\/$/, '') === there.pathname.replace(/\/$/, '') &&
    here.host === there.host
  );
};

const isSameHash = function(here, there) {
  return here.hash === there.hash;
};

class DynamicLoad {
  constructor() {
    this.load = this.load.bind(this);
    this.lastRoute = document.location.pathname + document.location.search;
    this.linkClickHandler = this.linkClickHandler.bind(this);
    this.didUpdateHandler = this.didUpdateHandler.bind(this);
    this.registerHandlers = this.registerHandlers.bind(this);
    this.popstateHandler = this.popstateHandler.bind(this);
  }

  load(url, pushState) {
    $('body').addClass('loading');

    return $.ajax({ type: 'GET', url: fullPath(url) })
      .then(html => {
        // Expose the jQuery object via an event for pre-render manipulation
        const $page = $('<div/>').append(
          $(html.replace('<!DOCTYPE html>', ''))
        );

        $(document).trigger('page.willUpdate', $page);

        // Update the title
        const $title = $page.find('title').eq(0);
        $('title').text($title.text());

        // Update each dynamic section
        $page.find('[data-dynamic-load]').each(function(i, el) {
          const $el = $(el);
          const slug = $el.data('dynamic-load');
          const content = $el.html();
          $(`[data-dynamic-load="${slug}"]`).html(content);
        });

        if (pushState) {
          window.history.pushState(null, document.title, fullPath(url));
          this.lastRoute = window.location.pathname + window.location.search;
        }

        $('body').removeClass('loading');

        $(document).trigger('page.didUpdate');
      })
      .catch(error => {
        window.location = fullPath(url);
      });
  }

  linkClickHandler(event) {
    // if (event.test) console.log(event);
    switch (true) {
      case isSamePage(window.location, event.currentTarget):
        if (isSameHash(window.location, event.currentTarget)) {
          event.preventDefault();
        }
      // Just follow the default behavior in these scenarios
      case event.ctrlKey:
      case event.metaKey:
      case isDifferentDomain(window.location, event.currentTarget):
        return Promise.resolve();
      // Load the content if this is a new page within the site.
      default:
        event.preventDefault();
        return this.load(event.currentTarget, true).then(() => {
          // Send reload a pageview
          logPageview();
        });
    }
  }

  didUpdateHandler(event) {
    // Handle vertical scroll position of new content
    if (window.location.hash) {
      const $el = $(window.location.hash).get(0);
      if ($el) $el.scrollIntoView();
    } else {
      $('.main-scroll').scrollTop(0);
    }

    // Collapse the mobile sidebar
    $('#sidebar').collapse('hide');
  }

  popstateHandler(event) {
    const curRoute = window.location.pathname + window.location.search;
    if (curRoute !== this.lastRoute) {
      this.load(window.location);
    }
  }

  registerHandlers() {
    $(document).on('click', '[data-dynamic-load] a', this.linkClickHandler);
    $(document).on('page.didUpdate', this.didUpdateHandler);
    $(window).on('popstate', this.popstateHandler);
  }
}

export default DynamicLoad;
