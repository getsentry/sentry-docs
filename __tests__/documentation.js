import { getJekyllData, printOnFail } from '../lib/helpers';
import { existsSync } from 'fs';
import path from 'path';

describe('Documentation', function() {
  beforeAll(() => {
    return getJekyllData().then(site => {
      this.site = site;
    });
  });

  test('must have a title', () => {
    const { documentation } = this.site;
    documentation.forEach(doc => {
      printOnFail(doc.path, () => {
        expect(doc.title).toBeTruthy();
      });
    });
  });

  test('must have an order number', () => {
    const { documentation } = this.site;
    documentation.forEach(doc => {
      const segments = doc.path.split('/');
      if (segments.length >= 2) return;
      printOnFail(doc.path, () => {
        expect(doc.sidebar_order).toBeGreaterThanOrEqual(0);
      });
    });
  });

  test('must match a category', () => {
    const { documentation, documentationCategories } = this.site;
    documentation.forEach(doc => {
      const segments = doc.path.split('/');
      if (segments.length > 3) return;
      const slug =
        segments.length === 3 ? segments[1] : segments[1].split('.')[0];
      const msg = `${doc.path}\n\nCategory "${slug}" does not exist`;
      printOnFail(msg, () => {
        expect(slug).toBeTruthy();
        const cat = documentationCategories.find(cat => cat.slug === slug);
        expect(cat).toBeTruthy();
      });
    });
  });

  // If nested in a folder, the folder name must match the name of a markdown
  // file in the same folder, which is used as the parent guide.
  //
  // Example:
  //
  //  _documentation/alert/made-up-category/document-a.md
  //  _documentation/alert/made-up-category.md
  test('must be in a folder that has a matching document', () => {
    const { documentation, documentationCategories } = this.site;
    documentation.forEach(doc => {
      const segments = doc.path.split('/').slice(0, -1);
      // We don't want to test top level docs, because they have categories
      if (segments.length <= 2) return;
      const dir = segments.join('/');
      const msg = `${doc.path}\n\nCategory "${dir}" does not exist`;
      printOnFail(msg, () => {
        expect(dir).toBeTruthy();
        const cat = documentation.find(doc => {
          return doc.path === `${dir}.md`;
        });
        expect(cat).toBeTruthy();
      });
    });
  });
});
