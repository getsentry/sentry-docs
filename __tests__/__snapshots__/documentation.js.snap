// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Documentation urls should not change unexpectedly 1`] = `
Array [
  "404/index.html",
  "_jest.json",
  "_platforms/_index.json",
  "_platforms/cocoa.json",
  "_platforms/cocoa/objc.json",
  "_platforms/cocoa/swift.json",
  "_platforms/cordova.json",
  "_platforms/csharp.json",
  "_platforms/csharp/aspnetcore.json",
  "_platforms/electron.json",
  "_platforms/elixir.json",
  "_platforms/go.json",
  "_platforms/go/http.json",
  "_platforms/java.json",
  "_platforms/java/android.json",
  "_platforms/java/appengine.json",
  "_platforms/java/log4j.json",
  "_platforms/java/log4j2.json",
  "_platforms/java/logback.json",
  "_platforms/java/logging.json",
  "_platforms/javascript.json",
  "_platforms/javascript/angular.json",
  "_platforms/javascript/angularjs.json",
  "_platforms/javascript/backbone.json",
  "_platforms/javascript/browser.json",
  "_platforms/javascript/ember.json",
  "_platforms/javascript/react.json",
  "_platforms/javascript/vue.json",
  "_platforms/minidump.json",
  "_platforms/node.json",
  "_platforms/node/connect.json",
  "_platforms/node/express.json",
  "_platforms/node/koa.json",
  "_platforms/objc.json",
  "_platforms/php.json",
  "_platforms/php/laravel.json",
  "_platforms/php/monolog.json",
  "_platforms/php/symfony.json",
  "_platforms/php/symfony2.json",
  "_platforms/python.json",
  "_platforms/python/aiohttp.json",
  "_platforms/python/asgi.json",
  "_platforms/python/bottle.json",
  "_platforms/python/celery.json",
  "_platforms/python/django.json",
  "_platforms/python/falcon.json",
  "_platforms/python/flask.json",
  "_platforms/python/pylons.json",
  "_platforms/python/pyramid.json",
  "_platforms/python/pythonawslambda.json",
  "_platforms/python/pythonserverless.json",
  "_platforms/python/rq.json",
  "_platforms/python/sanic.json",
  "_platforms/python/tornado.json",
  "_platforms/python/wsgi.json",
  "_platforms/ruby.json",
  "_platforms/ruby/rack.json",
  "_platforms/ruby/rails.json",
  "_platforms/rust.json",
  "accounts/membership/index.html",
  "accounts/pricing/index.html",
  "accounts/quotas/index.html",
  "accounts/require-2fa/index.html",
  "accounts/saml2/index.html",
  "accounts/sso/index.html",
  "api/auth/index.html",
  "api/events/delete-group-details/index.html",
  "api/events/delete-project-group-index/index.html",
  "api/events/get-group-details/index.html",
  "api/events/get-group-events-latest/index.html",
  "api/events/get-group-events-oldest/index.html",
  "api/events/get-group-events/index.html",
  "api/events/get-group-hashes/index.html",
  "api/events/get-group-tag-key-details/index.html",
  "api/events/get-group-tag-key-values/index.html",
  "api/events/get-project-event-details/index.html",
  "api/events/get-project-events/index.html",
  "api/events/get-project-group-index/index.html",
  "api/events/index.html",
  "api/events/put-group-details/index.html",
  "api/events/put-project-group-index/index.html",
  "api/index.html",
  "api/organizations/delete-organization-details/index.html",
  "api/organizations/get-event-id-lookup/index.html",
  "api/organizations/get-organization-details/index.html",
  "api/organizations/get-organization-index/index.html",
  "api/organizations/get-organization-projects/index.html",
  "api/organizations/get-organization-repositories/index.html",
  "api/organizations/get-organization-repository-commits/index.html",
  "api/organizations/get-organization-stats/index.html",
  "api/organizations/get-short-id-lookup/index.html",
  "api/organizations/index.html",
  "api/organizations/post-organization-index/index.html",
  "api/organizations/put-organization-details/index.html",
  "api/pagination/index.html",
  "api/permissions/index.html",
  "api/projects/delete-debug-files/index.html",
  "api/projects/delete-project-details/index.html",
  "api/projects/delete-project-key-details/index.html",
  "api/projects/delete-project-service-hook-details/index.html",
  "api/projects/get-debug-files/index.html",
  "api/projects/get-project-details/index.html",
  "api/projects/get-project-index/index.html",
  "api/projects/get-project-keys/index.html",
  "api/projects/get-project-service-hook-details/index.html",
  "api/projects/get-project-service-hooks/index.html",
  "api/projects/get-project-stats/index.html",
  "api/projects/get-project-tag-key-values/index.html",
  "api/projects/get-project-user-reports/index.html",
  "api/projects/get-project-users/index.html",
  "api/projects/index.html",
  "api/projects/post-debug-files/index.html",
  "api/projects/post-project-keys/index.html",
  "api/projects/post-project-service-hooks/index.html",
  "api/projects/post-project-user-reports/index.html",
  "api/projects/put-project-details/index.html",
  "api/projects/put-project-key-details/index.html",
  "api/projects/put-project-service-hook-details/index.html",
  "api/releases/delete-organization-release-details/index.html",
  "api/releases/delete-organization-release-file-details/index.html",
  "api/releases/delete-project-release-file-details/index.html",
  "api/releases/get-commit-file-change/index.html",
  "api/releases/get-organization-release-commits/index.html",
  "api/releases/get-organization-release-details/index.html",
  "api/releases/get-organization-release-file-details/index.html",
  "api/releases/get-organization-release-files/index.html",
  "api/releases/get-organization-releases/index.html",
  "api/releases/get-project-issues-resolved-in-release/index.html",
  "api/releases/get-project-release-commits/index.html",
  "api/releases/get-project-release-file-details/index.html",
  "api/releases/get-project-release-files/index.html",
  "api/releases/get-release-deploys/index.html",
  "api/releases/index.html",
  "api/releases/post-organization-release-files/index.html",
  "api/releases/post-organization-releases/index.html",
  "api/releases/post-project-release-files/index.html",
  "api/releases/post-release-deploys/index.html",
  "api/releases/put-organization-release-details/index.html",
  "api/releases/put-organization-release-file-details/index.html",
  "api/releases/put-project-release-file-details/index.html",
  "api/requests/index.html",
  "api/teams/delete-team-details/index.html",
  "api/teams/get-organization-teams/index.html",
  "api/teams/get-team-details/index.html",
  "api/teams/get-team-projects/index.html",
  "api/teams/get-team-stats/index.html",
  "api/teams/index.html",
  "api/teams/post-organization-teams/index.html",
  "api/teams/post-team-projects/index.html",
  "api/teams/put-team-details/index.html",
  "cli/configuration/index.html",
  "cli/dif/breakpad/index.html",
  "cli/dif/dsym/index.html",
  "cli/dif/elf/index.html",
  "cli/dif/index.html",
  "cli/dif/pdb/index.html",
  "cli/dif/proguard/index.html",
  "cli/index.html",
  "cli/installation/index.html",
  "cli/releases/index.html",
  "cli/send-event/index.html",
  "clients/cocoa/advanced/index.html",
  "clients/cocoa/dsym/index.html",
  "clients/cocoa/index.html",
  "clients/cocoa/migration/index.html",
  "clients/csharp/index.html",
  "clients/elixir/config/index.html",
  "clients/elixir/index.html",
  "clients/elixir/plug/index.html",
  "clients/elixir/usage/index.html",
  "clients/go/config/index.html",
  "clients/go/context/index.html",
  "clients/go/index.html",
  "clients/go/integrations/index.html",
  "clients/go/usage/index.html",
  "clients/index.html",
  "clients/java/agent/index.html",
  "clients/java/config/index.html",
  "clients/java/context/index.html",
  "clients/java/index.html",
  "clients/java/integrations/index.html",
  "clients/java/migration/index.html",
  "clients/java/usage/index.html",
  "clients/javascript/config/index.html",
  "clients/javascript/index.html",
  "clients/javascript/install/index.html",
  "clients/javascript/integrations/index.html",
  "clients/javascript/sourcemaps/index.html",
  "clients/javascript/tips/index.html",
  "clients/javascript/usage/index.html",
  "clients/node/coffeescript/index.html",
  "clients/node/config/index.html",
  "clients/node/index.html",
  "clients/node/integrations/index.html",
  "clients/node/sourcemaps/index.html",
  "clients/node/typescript/index.html",
  "clients/node/usage/index.html",
  "clients/perl/index.html",
  "clients/php/config/index.html",
  "clients/php/index.html",
  "clients/php/integrations/index.html",
  "clients/php/usage/index.html",
  "clients/python/advanced/index.html",
  "clients/python/api/index.html",
  "clients/python/breadcrumbs/index.html",
  "clients/python/index.html",
  "clients/python/integrations/index.html",
  "clients/python/platform-support/index.html",
  "clients/python/transports/index.html",
  "clients/python/usage/index.html",
  "clients/react-native/cocoapods/index.html",
  "clients/react-native/codepush/index.html",
  "clients/react-native/config/index.html",
  "clients/react-native/expo/index.html",
  "clients/react-native/index.html",
  "clients/react-native/manual-setup/index.html",
  "clients/react-native/ram-bundles/index.html",
  "clients/react-native/sourcemaps/index.html",
  "clients/ruby/breadcrumbs/index.html",
  "clients/ruby/config/index.html",
  "clients/ruby/context/index.html",
  "clients/ruby/index.html",
  "clients/ruby/install/index.html",
  "clients/ruby/integrations/index.html",
  "clients/ruby/processors/index.html",
  "clients/ruby/usage/index.html",
  "collections/_dev_components/includes/alert/index.html",
  "collections/_dev_components/includes/platform-content/index.html",
  "collections/_dev_components/includes/user-content/index.html",
  "collections/_dev_components/markdown-styleguide/links/index.html",
  "collections/_dev_components/tags/version-added/index.html",
  "data-management/data-forwarding/index.html",
  "data-management/rollups/index.html",
  "data-management/sensitive-data/index.html",
  "development/contribute/api/index.html",
  "development/contribute/contributing/index.html",
  "development/contribute/docs/index.html",
  "development/contribute/environment/index.html",
  "development/index.html",
  "development/sdk-dev/attributes/index.html",
  "development/sdk-dev/data-handling/index.html",
  "development/sdk-dev/features/index.html",
  "development/sdk-dev/index.html",
  "development/sdk-dev/interfaces/index.html",
  "development/sdk-dev/overview/index.html",
  "development/sdk-dev/unified-api/index.html",
  "enriching-error-data/breadcrumbs/index.html",
  "enriching-error-data/context/index.html",
  "enriching-error-data/environments/index.html",
  "enriching-error-data/scopes/index.html",
  "enriching-error-data/tracing/index.html",
  "enriching-error-data/user-feedback/index.html",
  "error-reporting/capturing/index.html",
  "error-reporting/configuration/draining/index.html",
  "error-reporting/configuration/filtering/index.html",
  "error-reporting/configuration/index.html",
  "error-reporting/quickstart/index.html",
  "error-reporting/security-policy-reporting/index.html",
  "history/index.html",
  "includes/alert/index.html",
  "includes/platform-content/index.html",
  "includes/user-content/index.html",
  "index.html",
  "ip-ranges/index.html",
  "license/index.html",
  "markdown-styleguide/links/index.html",
  "platforms/dotnet/aspnetcore/index.html",
  "platforms/dotnet/entityframework/index.html",
  "platforms/dotnet/index.html",
  "platforms/dotnet/log4net/index.html",
  "platforms/dotnet/microsoft-extensions-logging/index.html",
  "platforms/dotnet/nlog/index.html",
  "platforms/dotnet/serilog/index.html",
  "platforms/go/config/index.html",
  "platforms/go/echo/index.html",
  "platforms/go/fasthttp/index.html",
  "platforms/go/gin/index.html",
  "platforms/go/goroutines/index.html",
  "platforms/go/http/index.html",
  "platforms/go/index.html",
  "platforms/go/integrations/index.html",
  "platforms/go/iris/index.html",
  "platforms/go/martini/index.html",
  "platforms/go/migration/index.html",
  "platforms/go/negroni/index.html",
  "platforms/go/serverless/index.html",
  "platforms/go/transports/index.html",
  "platforms/index.html",
  "platforms/javascript/advance-settings/index.html",
  "platforms/javascript/angular/index.html",
  "platforms/javascript/cordova/index.html",
  "platforms/javascript/electron/index.html",
  "platforms/javascript/ember/index.html",
  "platforms/javascript/index.html",
  "platforms/javascript/ionic/index.html",
  "platforms/javascript/react/index.html",
  "platforms/javascript/sentry-testkit/index.html",
  "platforms/javascript/sourcemaps/index.html",
  "platforms/javascript/vue/index.html",
  "platforms/minidump/breakpad/index.html",
  "platforms/minidump/crashpad/index.html",
  "platforms/minidump/index.html",
  "platforms/node/connect/index.html",
  "platforms/node/default-integrations/index.html",
  "platforms/node/express/index.html",
  "platforms/node/index.html",
  "platforms/node/koa/index.html",
  "platforms/node/pluggable-integrations/index.html",
  "platforms/node/sourcemaps/index.html",
  "platforms/node/typescript/index.html",
  "platforms/php/default-integrations/index.html",
  "platforms/php/index.html",
  "platforms/php/laravel/index.html",
  "platforms/php/symfony/index.html",
  "platforms/python/aiohttp/index.html",
  "platforms/python/asgi/index.html",
  "platforms/python/aws_lambda/index.html",
  "platforms/python/bottle/index.html",
  "platforms/python/celery/index.html",
  "platforms/python/default-integrations/index.html",
  "platforms/python/django/index.html",
  "platforms/python/falcon/index.html",
  "platforms/python/flask/index.html",
  "platforms/python/gnu_backtrace/index.html",
  "platforms/python/index.html",
  "platforms/python/logging/index.html",
  "platforms/python/pyramid/index.html",
  "platforms/python/redis/index.html",
  "platforms/python/rq/index.html",
  "platforms/python/sanic/index.html",
  "platforms/python/serverless/index.html",
  "platforms/python/tornado/index.html",
  "platforms/python/wsgi/index.html",
  "platforms/rust/actix/index.html",
  "platforms/rust/env_logger/index.html",
  "platforms/rust/error_chain/index.html",
  "platforms/rust/failure/index.html",
  "platforms/rust/index.html",
  "platforms/rust/log/index.html",
  "platforms/rust/panic/index.html",
  "platforms/unrealengine/index.html",
  "project-resources/index.html",
  "server/beacon/index.html",
  "server/buffer/index.html",
  "server/changelog/index.html",
  "server/cli/cleanup/index.html",
  "server/cli/config/delete/index.html",
  "server/cli/config/discover/index.html",
  "server/cli/config/generate-secret-key/index.html",
  "server/cli/config/get/index.html",
  "server/cli/config/index.html",
  "server/cli/config/list/index.html",
  "server/cli/config/set/index.html",
  "server/cli/createuser/index.html",
  "server/cli/devserver/index.html",
  "server/cli/django/index.html",
  "server/cli/exec/index.html",
  "server/cli/export/index.html",
  "server/cli/files/get/index.html",
  "server/cli/files/index.html",
  "server/cli/files/info/index.html",
  "server/cli/help/index.html",
  "server/cli/import/index.html",
  "server/cli/index.html",
  "server/cli/init/index.html",
  "server/cli/permissions/add/index.html",
  "server/cli/permissions/index.html",
  "server/cli/permissions/list/index.html",
  "server/cli/permissions/remove/index.html",
  "server/cli/plugins/index.html",
  "server/cli/plugins/list/index.html",
  "server/cli/queues/index.html",
  "server/cli/queues/list/index.html",
  "server/cli/queues/purge/index.html",
  "server/cli/repair/index.html",
  "server/cli/run/cron/index.html",
  "server/cli/run/index.html",
  "server/cli/run/smtp/index.html",
  "server/cli/run/web/index.html",
  "server/cli/run/worker/index.html",
  "server/cli/shell/index.html",
  "server/cli/start/index.html",
  "server/cli/tsdb/index.html",
  "server/cli/tsdb/query/index.html",
  "server/cli/tsdb/query/organizations/index.html",
  "server/cli/upgrade/index.html",
  "server/config/index.html",
  "server/digests/index.html",
  "server/faq/index.html",
  "server/filestore/index.html",
  "server/inbound-mail/index.html",
  "server/index.html",
  "server/installation/docker/index.html",
  "server/installation/index.html",
  "server/installation/python/index.html",
  "server/integrations/github/index.html",
  "server/integrations/index.html",
  "server/internal-metrics/index.html",
  "server/monitoring/index.html",
  "server/nginx/index.html",
  "server/nodestore/index.html",
  "server/performance/index.html",
  "server/plugins/index.html",
  "server/queue/index.html",
  "server/sso/index.html",
  "server/throttling/index.html",
  "server/tsdb/index.html",
  "server/upgrading/index.html",
  "server/warnings/index.html",
  "sitemap.xml",
  "ssl/index.html",
  "support/index.html",
  "tags/version-added/index.html",
  "workflow/discover/index.html",
  "workflow/integrations/global-integrations/index.html",
  "workflow/integrations/index.html",
  "workflow/integrations/integration-platform/index.html",
  "workflow/integrations/integration-platform/ui-components/index.html",
  "workflow/integrations/legacy-integrations/index.html",
  "workflow/issue-owners/index.html",
  "workflow/notifications/alerts/index.html",
  "workflow/notifications/index.html",
  "workflow/notifications/notification-management/index.html",
  "workflow/notifications/workflow/index.html",
  "workflow/releases/index.html",
  "workflow/search/index.html",
  "workflow/visibility/index.html",
]
`;
